### PR TITLE
Add capacity api for some hochschul-canteens

### PR DESCRIPTION
**Pull request by Huber1**
We made an api for some canteens of the "Hochschule München". The API scheme is compatible with the currently present one on MENSA_GARCHING

### CODE REVIEW

File: src/entities.py [hunk 1]
```diff
@@ -199,7 +199,7 @@ def __init__(self, long_name: str, location: Location, url_id: int, queue_status
         "Mensa Pasing",
         Location("Am Stadtpark 20, München", 48.141568, 11.451119),
         432,
-        None,
+        "https://beta.api.betterhm.app/v1/capacity/MENSA_PASING",
```
Comment (COM8) [inline src/entities.py:202]: The API is clearly marked as beta. Is this intended?

**Review by COM8 — COMMENTED** [review]
Thanks for contributing! Only one question

File: src/entities.py (hunk 1 above)
Comment (Huber1) [reply src/entities.py:202]: we have some other stuff there too, but we'll change the domain soon and remove the beta.
so yes, it is intended

**Comment by Huber1** [comment]
Hey,
just wanted to check if there are any issues with the request. I'm happy to make changes if needed

**Pull request merged**